x-pack/plugins/security/public/session/session_expired.test.ts

```diff
@@ -4,11 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { coreMock } from 'src/core/public/mocks';
 import { SessionExpired } from './session_expired';
 
-describe('Session Expiration', () => {
+describe('#logout', () => {
```

Comment:
These unit tests didn't really make sense anymore as written. I took this opportunity to rewrite them all.